### PR TITLE
updated send_to_device, for ElasticArray type.

### PR DESCRIPTION
**Pull request by norci**

### CODE REVIEW

File: src/utils/device.jl [hunk 1]
```diff
@@ -22,6 +23,8 @@ send_to_device(
             <:Any,
             <:Base.ReshapedArray{<:Any,<:Any,<:SubArray{<:Any,<:Any,<:CircularArrayBuffer}},
         },
+        SubArray{<:Any,<:Any,<:ElasticArray,<:Any,<:Any},
+        ElasticArray,
```
Comment (findmyway) [inline src/utils/device.jl:27]: Is it necessary here? I remember that ElasticArray supports CUDA.jl but I may be wrong. Away from computer so I can't verify it…

Comment (norci) [reply src/utils/device.jl:27]: I'm not sure.
I got an error, and filed an issue https://github.com/JuliaArrays/ElasticArrays.jl/issues/30 .

Comment (findmyway) [reply src/utils/device.jl:27]: 👍
I'll add a monkey patch here if it's still not fixed until next week.

Comment (findmyway) [reply src/utils/device.jl:27]: For `ElasticArray` with `data` of `CuArray`, there's no need to create a new copy with `CuArray(x)`. But I think that's a rare case here.

**Pull request merged**